### PR TITLE
Move tests of ecdsa to signing package

### PR DESCRIPTION
**Pull request by feihujiang**
Signed-off-by: feihujiang <jiangfeihu@huawei.com>

### CODE REVIEW

File: bin/run_tests [hunk 1]
```diff
@@ -13,6 +13,12 @@ cd core
 nose2 -v
 cd ..
 
+PYTHONPATH=$top_dir/signing
+export PYTHONPATH
```
Comment (dcmiddle) [inline bin/run_tests]: Think you might be missing this before exporting pythonpath
PYTHONPATH=$PYTHONPATH:$top_dir/signing/build/lib.linux-x86_64-2.7

**Comment by feihujiang** [comment]
@dcmiddle I updated this PR, please have a look.

**Comment by feihujiang** [comment]
@dcmiddle After this PR merged, I'll do code refactoring about [generate_identifier(signingkey)](https://github.com/hyperledger/sawtooth-core/blob/master/validator/gossip/signed_object.py#L64), [generate_signing_key(wifstr=None)](https://github.com/hyperledger/sawtooth-core/blob/master/validator/gossip/signed_object.py#L77) and [get_verifying_key(serialized_msg, serialized_sig)](https://github.com/hyperledger/sawtooth-core/blob/master/validator/gossip/signed_object.py#L93) in SignedObject, move all of them to signing module, What's your point?

**Comment by dcmiddle** [comment]
Looks like the PR is failing Pep-8 checks. The original code predated the lint rules.
You can either update the code to pass the lint checking (bin/run_lint) by 
1. excluding it (all the other tests appear exempted right now)
or 
2. update the code to make it comply

To exclude, add a line to bin/run_lint around line 41 like 
grep -v signing/tests

**Comment by feihujiang** [comment]
@dcmiddle I updated the code to make it comply. please review.

**Comment by feihujiang** [comment]
@dcmiddle I just rebased this PR, could you please take a look?

**Pull request merged**